Resources/Maps/_RMC14/lv624.yml

```diff
@@ -142419,216 +142581,6 @@ entities:
     - type: Transform
       pos: -6.5,-96.5
       parent: 1
-- proto: FoamedAluminiumMetal
```

Comment:
Can't tell from the YAML where these were but these should be there in 13
![image](https://github.com/user-attachments/assets/c4c2dce2-55eb-4fad-82d7-8f0523c0d665)